gettingStartedWithAstraBlock/README.md

```diff
@@ -1,5 +1,5 @@
 # Overview
-Stream real-time, human-readable blockchain data to your app with Astra Block.
+Stream real-time, human-readable blockchain data to your app with Astra Block. Simply request access to the feature within Astra (we'll enable it for your account within 24 hours) and you can instantly create a blockchain database that auto updates as new blocks are mined.
```

Comment:
```suggestion
Stream real-time, human-readable blockchain data to your app with Astra Block. Simply request access to the feature within Astra (we'll enable it for your account within 24 hours) and you can create a blockchain database that auto updates as new blocks are mined.
```
(I removed "instantly" because it is confusing how it take 24 hours to enable an account but you can create a blockchain instantly.)